CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
+- Update `installpsh-suse.sh` to work with the `tar.gz` package. (#5309)
+- Add `Jobject` serialization support to `ConvertTo-Json`. (#5141)
+- Display full help with 'help' function. (#5195) (Thanks @rkeithhill!)
+- Fix `help` function to not pipe to `more` if objects are returned instead of help text. (#5395)
+- Fix `Unblock-File` to not write an error if the file is already unblocked. (#5362) (Thanks @iSazonov!)
+- Add error on legacy credential over non-HTTPS for Web Cmdlets. (#5402) (Thanks @markekraus!)
+- Clean up FullCLR code from Web Cmdlets. (#5376) (Thanks @markekraus!)
+- Exclude cmdlets that are not supported on Unix platforms. (#5083)
+- Make `Import-Csv` support `CR`, `LF` and `CRLF` as line delimiters. (#5363) (Thanks @iSazonov!)
+- Fix spelling in Web Cmdlet errors. (#5427) (Thanks @markekraus!)
+- Add `SslProtocol` support to Web Cmdlets. (#5329) (Thanks @markekraus!)
+- Fix single value JSON `null` in `Invoke-RestMethod`. (#5338) (Thanks @markekraus!)
+- Add `PSTypeName` Support for `Import-Csv` and `ConvertFrom-Csv`. (#5389) (Thanks @markekraus!)
+
+### Build and Packaging Improvements
+
+- Use `RCEdit` to embed icon and version information into `pwsh.exe`. (#5178)
+- Update Docker file for Nano Server 1709 release. (#5252)
+- Change VSCode build task to use `pwsh`. (#5255)
+- Refactor building and packaging scripts for signing in release build workflow. (#5300)
+- Always build with `-CrossGen` in CI to verify a fix in `CrossGen` tool. (#5315)
+- Separate `Install-PowerShellRemoting.ps1` from `psrp.windows` nuget package. (#5330)
+- Include symbols folder an embedded zip when packaging symbols. (#5333)
+- Add Uniform Type Identifier conforming with Apple standards using a reverse DNS style prefix. (#5323)
+- Update `Wix` toolset download link to newer version 3.11 (#5339) (Thanks @bergmeister!)
+- Re-enable macOS launcher after fixing an issue that blocked macOS package generation. (#5291) (Thanks @thezim!)
+- Set expected binaries and variable name for folder for symbols build. (#5357)
+- Rename and update PowerShell `ETW` manifest to remove the Windows PowerShell dependency. (#5360)
+- Add ability to produce `tar.gz` package for Raspbian. (#5387)
+- Update `Find-Dotnet` to find dotnet with the compatible SDK. (#5341) (Thanks @rkeithhill!)
+- Add signing manifest and script to update it with production values. (#5397)
+- Add `install-powershell.ps1` to install PowerShell Core on windows. (#5383)
+- Make `-Name` a dynamic parameter in `Start-PSPackage`. (#5415)
+- Support `[package]` tag in PR CI and fix nightly build on macOS. (#5410)
+- Enhance `install-powershell.ps1` to work on Linux and macOS. (#5411)
+- Move the `RCEdit` step to the build phase rather than the packaging phase. (#5404)
+- Allow packaging from a zip package to allow for signing. (#5418)
+- Add automation to validate PowerShell Core packages using Docker containers. (#5401)
+- Try fixing the `brew update` issue in bootstrap script. (#5400)
```

Comment:
Can we remove `Try` and just say `Fixed`?

Reply:
Fixed.